rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -2035,7 +2035,7 @@ def test_upgrade_db_40_to_41(user_data_dir):
             name VARCHAR[30] NOT NULL PRIMARY KEY,
             api_key TEXT
             )""") is True
-        assert cursor.execute('SELECT * FROM external_service_credentials').fetchall() == [('etherscan', 'LOL'), ('blockscout', 'LOL2')]  # noqa: E501
+        assert cursor.execute('SELECT * FROM external_service_credentials').fetchall() == [('etherscan', 'LOL'), ('blockscout', 'LOL2'), ('covalent', 'lollol')]  # noqa: E501
```

Comment:
you also need to assert it's no longer there after the upgrade

Reply:
It happens here https://github.com/rotki/rotki/pull/7316/files#diff-dc14676416c357a3a363ef102d35370ee0bd2ebc98976e6509f3e4805a12e87fR2088

![imagen](https://github.com/rotki/rotki/assets/5068010/18da88bb-a09c-48d6-8fba-1262fbb06343)

Reply:
ah well lol ... yeah you are right. Thanks for the screenshot cause link does not work in PR reviews